Dockerfile

```diff
@@ -21,7 +25,7 @@ COPY . .
 
 EXPOSE 5004
 
-CMD ["gunicorn", "--bind=0.0.0.0:5004", "chickadee.wsgi:application"]
+CMD gunicorn --bind=0.0.0.0:5004 --timeout 90 chickadee.wsgi:application
```

Comment:
What is the default timeout length, 30?

Reply:
I know nothing about `gunicorn`. Maybe @cairosanders could answer your question since I copied the Dockerfile that @cairosanders shared this morning. ?

Reply:
default is 30 ya